internal/docstore/mongodocstore/localmongo.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+# Copyright 2019 The Go Cloud Development Kit Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Runs only tests relevant to the current pull request.
+# At the moment, this only gates running the Wire test suite.
+# See https://github.com/google/go-cloud/issues/28 for solving the
+# general case.
+
+# https://coderwall.com/p/fkfaqq/safer-bash-scripts-with-set-euxo-pipefail
+set -euo pipefail
+
+echo "Starting MongoDB..."
+docker rm -f mongo 2> /dev/null || :
```

Comment:
Instead of killing a previous Mongo process, you could change the `docker run` line to include `--rm`, which will automatically remove it once it is stopped. (Here and elsewhere.)

Reply:
I actually want this to stop and restart all of the processes. "automatically remove it once it is stopped" implies someone is stopping it via `docker` explicitly, which may not happen.